scripts/publish-vsix.js

```diff
@@ -12,16 +12,15 @@ if (!vsix.length) {
   shell.exit(1);
 }
 
-const vsce = '../../node_modules/.bin/vsce';
 const VSCE_PERSONAL_ACCESS_TOKEN = process.env['VSCE_PERSONAL_ACCESS_TOKEN'];
 let vscePublish = '';
 if (VSCE_PERSONAL_ACCESS_TOKEN) {
   vscePublish = shell.exec(
-    `${vsce} publish --pat ${VSCE_PERSONAL_ACCESS_TOKEN} --packagePath ${vsix}`
+    `vsce publish --pat ${VSCE_PERSONAL_ACCESS_TOKEN} --packagePath ${vsix}`
```

Comment:
@sfsholden the `--pat` attribute takes care of using the token so we won't need to run `vsce login` in a clean environment, we'll only need to define the token as an env variable.